interface/patient_file/history/history_sdoh.php

```diff
@@ -0,0 +1,293 @@
+                                    <?php
+                                    $res = sqlStatement("SELECT `id`, `username`, CONCAT(`fname`, ' ', `lname`) as name FROM `users` WHERE `authorized`=1");
+                                    while ($orow = sqlFetchArray($res)) {
+                                        echo " <option value='" . attr($orow['name']) . "'";
+                                        if ($info['assessor'] == $orow['name']) {
+                                            echo " selected";
+                                        }
+                                        echo ">" . text($orow['name']) . "</option>\n";
+                                    }
+                                    ?>
+                                </select>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <!-- Domains + Context in 2-col grid -->
+                <div class="row">
+                    <?php
+                    $domains = [
+                        ['key' => 'food_insecurity',           'label' => xlt('Food Insecurity'),           'list' => 'sdoh_food_insecurity_risk'],
+                        ['key' => 'housing_instability',       'label' => xlt('Housing Instability'),       'list' => 'sdoh_housing_worry'],
+                        ['key' => 'transportation_insecurity', 'label' => xlt('Transportation Insecurity'), 'list' => 'sdoh_transportation_barrier'],
+                        ['key' => 'utilities_insecurity',      'label' => xlt('Utilities Insecurity'),      'list' => 'sdoh_utilities_shutoff'],
+                        ['key' => 'interpersonal_safety',      'label' => xlt('Interpersonal Safety'),      'list' => 'sdoh_ipv_yesno'],
+                        ['key' => 'financial_strain',          'label' => xlt('Financial Strain'),          'list' => 'sdoh_financial_strain'],
+                        ['key' => 'social_isolation',          'label' => xlt('Social Connections / Isolation'),'list' => 'sdoh_social_isolation_freq'],
+                        ['key' => 'childcare_needs',           'label' => xlt('Childcare Needs'),           'list' => 'sdoh_childcare_needs'],
+                        ['key' => 'digital_access',            'label' => xlt('Digital Access'),            'list' => 'sdoh_digital_access'],
+                    ];
+                    foreach ($domains as $d) :
+                        $k       = $d['key'];
+                        $label   = $d['label'];
+                        $list_id = $d['list'];
+                        $val     = v($info, $k);
+                        $notes   = v($info, $k . '_notes');
+                        ?>
+                        <div class="col-12 col-md-6">
+                            <div class="card h-100 mb-2">
+                                <div class="card-header font-weight-bold"><?php echo text($label); ?></div>
```

Comment:
looks like you are double escaping $label. Rec changing the xlt to xl in your label in above domains array.